icubam/www/handlers/db.py

```diff
@@ -50,6 +52,13 @@ def initialize(self, upload_path, config, db_factory):
 
   @tornado.web.authenticated
   def get(self, collection):
+    if self.current_user.acces_type not in self.GET_ACCESS:
+      logging.debug(
+        f"API called with incorrect access_type: {self.current_user.access_type}."
```

Comment:
That's from  https://github.com/icubam/icubam/pull/295, right, and so we need to first merge that one?